ui/package.json

```diff
@@ -14,14 +14,14 @@
     "precommit": "lint-staged"
   },
   "lint-staged": {
-    "gitDir": "../",
```

Comment:
do you not need `gitDir` anymore?

Reply:
Nope! lint-staged walks up the directory looking for a `.git` dir now: https://github.com/okonet/lint-staged/issues/271

Reply:
Awesome 👌